tests/test_ensembl.py

```diff
@@ -4,4 +4,4 @@
 # Make sure we don't update the release without realising it.
 def test_version():
     release = stdpopsim.catalog.ensembl_info.release
-    assert release == 103
+    assert release == 111
```

Comment:
Ack, this thing.  See https://github.com/popsim-consortium/stdpopsim/issues/1521

Reply:
Please don't change this here.

Reply:
Ok, so best to just let this test fail until the solution is worked out? I tried for follow the threads and it seems like the PR #1536 is still open?

Reply:
Yes, you should definitely not change this until this is actually fixed. (However, that'd be a great thing to fix next...)